.travis.yml

```diff
@@ -3,7 +3,11 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.1.0
   - export PATH="$HOME/.yarn/bin:$PATH"
   - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 --no-sandbox --disable-dev-shm-usage http://localhost &
-branches:
+install:
```

Comment:
My understanding is that `assertNoAxeViolations` in e2e requires build files to work, as they check actual pages in `localhost:3001`. Previously build files were in the branch, so this config file adds the extra `yarn build` step and fixes that.

additional note, since the build files were previously only updated on releases, I think the old config here was a bit broken so its nice to fix that